From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 18 Dec 2013 17:16:33 +0100
Subject: loginctl: when showing device tree of seats with no devices show
 something useful

(cherry picked from commit ddae67fa1bdd86b480a60ad6fc1d90d6f35a03d0)
---
 src/login/sysfs-show.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/login/sysfs-show.c b/src/login/sysfs-show.c
index f7d11dd..cb12ebf 100644
--- a/src/login/sysfs-show.c
+++ b/src/login/sysfs-show.c
@@ -181,6 +181,8 @@ int show_sysfs(const char *seat, const char *prefix, unsigned n_columns) {
         first = udev_enumerate_get_list_entry(e);
         if (first)
                 show_sysfs_one(udev, seat, &first, "/", prefix, n_columns);
+        else
+                printf("%s%s%s\n", prefix, draw_special_char(DRAW_TREE_RIGHT), "(none)");
 
         return r;
 }
